accounts/templates/account/local_user_exists.html

```diff
@@ -10,17 +10,16 @@
 <div class="container">
   <div class="jumbo" style="padding-top: 25px">
       <h2 style="font-size: 48px; text-align: center; font-weight: 300;">
-          Login Unsuccessful
+          {% trans "Login Unsuccessful" %}
       </h2>
       <h3 style="text-align: center; font-weight: 300;">
-        A Lookit study participant exists with the email address for your OSF user account.
-        Create an OSF user with a different email address and try logging into Experimenter with that.
+        {% trans "A Lookit study participant exists with the email address for your OSF user account. Create an OSF user with a different email address and try logging into Experimenter with that. " %}
       </h3>
       <h3 style="text-align: center; font-weight: 300;">
-          If this should not have occurred and the issue persists, please report it to <a href="mailto:support@osf.io">support@osf.io</a>.
+          {% trans "If this should not have occurred and the issue persists, please report it to <a href="mailto:support@osf.io">support@osf.io" %}</a>.
```

Comment:
should this include the closing tag or not include the link at all?

Reply:
I've put the closing tag inside the {% trans } and made the inner quotes single.

`
{% trans "If this should not have occurred and the issue persists, please report it to <a href='mailto:support@osf.io'>support@osf.io</a>" %}.
`